File: ex5/ccpp/ex05.c
#include <stdio.h>
#include <stdlib.h>

int main(int argc, char *argv[]){

    float salarioMinimo, salario, quantSalario;

    printf("Entre com o seu salário: ");
    scanf("%f", &salario);    

    printf("Entre com o salário mínimo: ");
    scanf("%f", &salarioMinimo);

    quantSalario = salario / salarioMinimo;

    printf("Ele recebe: %5.2f \n\n", quantSalario); 

    return 0;
}

File: ex5/java/ex05.java
package ex5.java;

import javax.swing.JOptionPane;

public class ex05 {
    public static void main(String[] args) {
        
        float salario, salarioMinimo, quantSalario;

        salario = Float.parseFloat(JOptionPane.showInputDialog(null, "Entre com o seu salário"));
        salarioMinimo = 788.00f;



        quantSalario = salario / salarioMinimo;
        
        JOptionPane.showMessageDialog(null, "Ele recebe: " + String.format("%.2f", quantSalario) + " salárioMinimo");

    }
}
